units/crons_remote_five_minute.go

```diff
@@ -51,6 +51,7 @@ func (j *cronsRemoteFiveMinuteJob) Run(ctx context.Context) {
 		PopulateActivationJobs(10),
 		PopulateHostProvisioningConversionJobs(j.env),
 		PopulateHostRestartJasperJobs(j.env),
+		checkGithubLimit(j.env),
```

Comment:
is it worth doing this every minute instead of every 5? Is checking the limit free?

Reply:
It is free but everything below a minute is done by creating a job on amboy and i don't think that makes sense for logging these limits. 

But i think this more than enough because the splunk alert only runs every 10 minutes so we would have enough information to alert on